doc/Makefile

```diff
@@ -30,11 +31,15 @@ all: aspect.tag manual.pdf
 	@perl manual/cookbooks/annotate.pl $< >$@
 	@for i in `seq 1 10`; do sed -i 's/{\([^!]*\)!\([^!]*\)!\([^!]*\)!\([^}]*\)}/{\1!\2!\3\/\4}/' $@; done
 
+# Generate changes.h from the bits in modules/changes/
+modules/changes.h: modules/changes/* modules/create_changes.sh modules/current_changes_*
```

Comment:
Ah, fantastic -- it works in targest and prerequisites. I had only remembered the case where it is used elsewhere, such as in the last example at https://www.gnu.org/software/make/manual/html_node/Wildcard-Examples.html#Wildcard-Examples

In general, wildcards for prerequisites is dangerous because it only picks up files *that are already in the directory*. It won't pick up files that are created by another rule, and because it doesn't pick them up, they won't be generated. In your case, none of the listed prerequisites fall in this category, so this is not an issue.